src/_guides/language/effective-dart/usage.md

```diff
@@ -1128,11 +1122,33 @@ expression inside:
 * A switch case expression&mdash;the part right after `case` before the `:`, not
   the body of the case.
 
-(Default values are *not* included in this list because future versions of Dart
+(Default values are not included in this list because future versions of Dart
 may support non-const default values.)
 
-Basically, any place where it would be a static error to write `new` instead of
-`const`, you can and should omit the `const`.
+Basically, any place where it would be an error to write `new` instead of
+`const`, Dart 2 allows you to omit the `const`. If your code does not need to
```

Comment:
does not -> doesn't

https://developers.google.com/style/contractions

Reply:
Removed the sentence.